src/components/BarList/BarLabel.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC, ReactNode } from 'react'
+import { composeClasses } from 'lib/classes'
+import { Size } from 'interfaces'
+
+interface IPropsBarLabel {
+  children: ReactNode
+  fontSizeBar?: Size
+  href: string | undefined
+  textBarColor?: string
+}
+
+export const BarLabel: FC<IPropsBarLabel> = ({
+  children,
+  fontSizeBar,
+  href,
+  textBarColor
+}) => {
+  if (href) {
+    return (
+      <a
+        href={href}
+        rel="noreferrer"
+        style={{ color: textBarColor }}
+        data-testid="label-link"
+        className={composeClasses(
+          'hover:underline',
+          fontSizeBar && `text-${fontSizeBar}`
+        )}
+        target="_blank"
+      >
+        {children}
+      </a>
+    )
+  }
+
+  return (
+    <p
+      data-testid="label-text"
+      style={{ color: textBarColor }}
+      className={fontSizeBar && `text-${fontSizeBar}`}
```

Comment:
Actually, I was noting that I did an unnecessary validation since it will always have a default value. I will replace the p with the Text component and use the prop size like this:

`<Text size={fontSizeBar} >....`